_posts/2012-11-16-november-2012.md

```diff
@@ -20,13 +20,13 @@ To support you in collecting and reporting your site [search metrics]({{ site.ba
 1. **Top changing search terms (movement up/down).** Shows trending topics, what's hot or not. Use this report to investigate why people might be losing interest in once-popular content and consider archiving if it's no longer needed. For newly popular terms, create new content or update existing content to ensure it's current, accurate, and complete. 
 1. **Top clicked URLs for specific queries.** The most common results that searchers found most relevant or appealing, for specific search terms. Review these popular pages and other customer top tasks to make sure they are current, accurate, complete and compelling.
 
-### 2. Recommended Pages in Type-ahead Suggestions
+### 2. Recommended Pages in typeahead Suggestions
```

Comment:
Should this be title case?

Reply:
Yes, but this file was deleted with the other PR, so I didn't bother cleaning it up after the global edit.